.mailmap

```diff
@@ -216,8 +216,9 @@ Duane Nykamp <nykamp@umn.edu>
 Mihai A. Ionescu <ionescu.a.mihai@gmail.com>
 Sarwar Chahal <chahal.sarwar98@gmail.com>
 Hamish Dickson <hamish.dickson@gmail.com>
-Peter Brady <petertbrady@gmail.com>
-Peter Brady <petertbrady@gmail.com> <ptb@lanl.gov>
+Peter Brady <petertbrady@gmail.com> Peter Brady <ptb@.lanl.gov>
+Peter Brady <petertbrady@gmail.com> Peter Brady <ptb@lanl.gov>
```

Comment:
@pbrady should confirm which email he would prefer to have in the AUTHORS file.

Reply:
the gmail one is fine.